crates/wasm-smith/src/core.rs

```diff
@@ -568,6 +570,8 @@ impl Module {
                     return false;
                 }
             },
+
+            _ => todo!(),
```

Comment:
Mind filling this out to avoid the `_`? For example `(HT::Exact(_), _) | (_, HT::Exact(_)) => todo!()` or something like that